.github/workflows/pre-commit.yml

```diff
@@ -53,6 +53,12 @@ jobs:
             all:
               - added|modified: '**/*'
 
+      - uses: getsentry/action-setup-volta@e4939d337b83760d13a9d7030a6f68c9d0ee7581 # v2.0.0
+
+      - name: Install node dependencies
+        run: |
+          yarn install --frozen-lockfile
```

Comment:
@scttcper I wonder if there's some way we could say "Just install what's needed for linting"

Reply:
Yeah, I already tried very hard to do that with the v1 PR. Couldn't figure it out. 25 seconds from warm cache is fine though, I realized it wasn't worth it to keep going.